sql/catalyst/src/main/java/org/apache/spark/sql/catalog/v2/expressions/Expressions.java

```diff
@@ -96,7 +96,7 @@ public Transform bucket(int numBuckets, String... columns) {
    * @param column an input column
    * @return a logical identity transform with name "identity"
    */
-  public Transform identity(String column) {
+  public static Transform identity(String column) {
```

Comment:
Fixing these methods that should have been static is required here: https://github.com/apache/spark/pull/24246/files#diff-d921fee79b4e63ab684dec9d10afeb44R63.